src/playwright/browser.ts

```diff
@@ -1,20 +1,16 @@
-import { BrowserContext } from 'playwright';
-import { TypedEmitter } from 'tiny-typed-emitter';
+import { EventEmitter } from 'events';
+import { BrowserContext, Browser as PlaywrightBrowser } from 'playwright';
 
 export interface BrowserOptions {
     browserContext: BrowserContext;
     version: string;
 }
 
-export interface BrowserEvents {
-    disconnected: () => void;
-}
-
 /**
  * Browser wrapper created to have consistent API with persistent and non-persistent contexts.
  */
-export class Browser extends TypedEmitter<BrowserEvents> {
-    browserContext: BrowserContext;
+export class Browser extends EventEmitter implements PlaywrightBrowser {
+    private _browserContext: BrowserContext;
```

Comment:
It's used in a test. It's prefixed with `_` otherwise [it would be public](https://www.typescriptlang.org/play?#code/MYGwhgzhAEAa0G8BQ1XQA4CcCWA3MALgKbTEQEBc0AdgK4C2ARkZtALzQCMA3EgL5A) in plain JS.

Reply:
> otherwise it would be public in plain JS.

it will be public anyway, [right](https://www.typescriptlang.org/play?#code/MYGwhgzhAEAa0G8BQ1XQA4CcCWA3MALgKbQD6xEBAXNAHYCuAtgEZGbQC80AjANxIBfIA)?

Reply:
Yep, but the underscore indicates it's internal use only.

Reply:
yes, in some IDE. just like the `.d.ts` file will, regardless of the name, or not?

Reply:
https://stackoverflow.com/questions/4484424/is-the-underscore-prefix-for-property-and-method-names-merely-a-convention

Reply:
i dont understand how that is related to what i just said. this is library, it will be published together with typings, the information will be there for editors. as the answers as saying, it is just a convention, without any real effect.

anyway, as said, i am fine with this, just wanted to understand the motivation

Reply:
> just like the .d.ts file will, regardless of the name, or not?

It will be accessible via `browser._browserContext` like other many internal Node.js properties.

Reply:
Also there was already `_version` and `_isConnected`.